src/peer.js

```diff
@@ -130,6 +130,7 @@ class Peer extends EventEmitter {
    * @param {Object} [options] - Optional arguments for DataConnection.
    * @param {string} [options.connectionId] - An ID to uniquely identify the connection.
    * @param {string} [options.label] - Label to easily identify the connection on either peer.
+   * @param {Object} [options.dcInit] - Config object works as a RTCDataChannelInit.
```

Comment:
I think we should define the possible options? (We use these as a base for creating the documents).
`ordered`, `maxPacketLifeTime`, and `maxRetransmits`. I think the other options for `createDataChannel` aren't useful for skyway so can be left out.

Reply:
👍 
But may it be enough to add a link to [spec](https://www.w3.org/TR/webrtc/#dom-rtcdatachannelinit)?

Reply:
> But may it be enough to add a link to spec?

Sounds good. Maybe a note saying browser implementations may vary?